Documentation/Sources/configure-extension.md

```diff
@@ -0,0 +1,71 @@
+# Configure the Notification Content extension
+
+With the AEPNotificationContent package now available after following the [installation steps](./getting-started.md), the app must now be configured to use the Adobe SDK when the Notification Content extension is called.
+
+## App configuration
+
+In your `AppDelegate`, after the user has granted your app permission to display notifications, create any custom notification actions needed by your app and register a category with identifier `AEPNotification`.
+
+```swift
+let center = UNUserNotificationCenter.current()
+
+// ask user for permission to display notifications
+center.requestAuthorization(options: [.badge, .sound, .alert]) { [weak self] granted, _ in
+    
+    // return early if the user does not consent 
+    guard granted else { return }
+    
+    center.delegate = self
+    
+    // these `UNNotificationAction`s are for demonstration purposes only
+    let openInAppAction = UNNotificationAction(identifier: "customAction", title: "Open in App")
+    let replyAction = UNNotificationAction(identifier: "replyAction", title: "Reply")
```

Comment:
My suggestion here will be to remove these actions associated with this notification category.
1. We don't have any predefined actionID and that will be handled our NotificationContentExtension
2. Reply action will be confusing since we done support replying via notification just yet